c/shared/source/uforead/uforead.c

```diff
@@ -1255,23 +1266,17 @@ static int findGLIFRecByName(ufoCtx h, char *glyphName)
     return -1;
 }
 
-static void updateGLIFRec(ufoCtx h, int state) {
+static void updateGLIFRec(ufoCtx h, char* glyphName, char* fileName) {
     int index;
-    char *glyphName;
-
-    glyphName = h->parseKeyName;
 
     index = findGLIFRecByName(h, glyphName);
     if (index == -1) {
         message(h, "Warning: glyph '%s' is in the processed layer but not in the default layer.", glyphName);
-        getKeyValue(h, "</string>", state); /* consume the file name */
     } else {
-        char* fileName;
         GLIF_Rec* glifRec;
 
         glifRec = &h->data.glifRecs.array[index];
 
-        fileName = getKeyValue(h, "</string>", state);
         if (fileName == NULL) {
             fatal(h, ufoErrParse, "Encountered glyph reference in alternate layer's contents.plist with an empty file path. Text: '%s'.", getBufferContextPtr(h));
```

Comment:
Curious to hear your thoughts about this — a key with an empty file path as its value will not fail anymore because the new libxml2 parsing code will simply skip a key that has a missing or NULL value, rather than fail like it did in the older parser code.

I think this is fine but wanted to hear if you have concerns about it. I didn't think of adding warnings since we're trying to reduce the amount of warning messages we have, but I can add them in this case if needed.